crates/neon/src/types_impl/extract/with.rs

```diff
@@ -48,4 +58,21 @@ where
     }
 }
 
+impl<'cx, F, O> TryIntoArgumentsInternal<'cx> for With<F, O>
```

Comment:
Nit, it makes a bit more sense to me for this implementation to be with the `TryIntoArguments` code rather than here, but if you think it's better here, I'm okay with that.

Reply:
I moved it to `function/mod.rs` with the other impls.